src/test/resources/application.yml

```diff
@@ -0,0 +1,8 @@
+spring:
```

Comment:
Not able to connect to mongodb locally. Will need a fix. Opened a new issue to track.